Plugins/Player/NWScript/nwnx_player.nss

```diff
@@ -427,6 +427,13 @@ object NWNX_Player_GetGameObject(object oPlayer);
 /// @param nMask A mask of OBJECT_UI_DISCOVERY_*, or -1 to clear the override
 void NWNX_Player_SetObjectUiDiscoveryMaskOverride(object oPlayer, object oObject, int nMask);
 
+/// @brief Send a party invite from oInviter to oToPlayer
+/// @param oToPlayer The player to invite
+/// @param oInviter The one inviting the player
+/// @param bForceInvite TRUE: Sends the invite even if the target ignores invites
+/// @param bHideDialog TRUE: Does not show the party invitation dialog
```

Comment:
If you hide the dialog, how do they accept/reject the invite? Through the playerlist window?

Reply:
Yes, that was the intention

Reply:
Ok! I assume you tested it 😛

Reply:
Yes, it works!